app/views/admin/users/index.html.erb

```diff
@@ -0,0 +1,10 @@
+<h1><%= t(".title") %></h1>
+<% @users.each do |user| %>
+  <ul id="user_<%= user.id %>">
+    <li><%= t(".name") %>: <%= user.name %></li>
```

Comment:
Have a look at Rails' `human_attribute_name` rather than defining the translation for these attributes under `users.index.name` -- because the former can be reused easier in more places

Ref: http://guides.rubyonrails.org/i18n.html#translations-for-active-record-models

Reply:
I will take a look and implement in another PR, thanks Elle